Runtime/gestures/mouse_tracking.cs

```diff
@@ -4,23 +4,43 @@
 using Unity.UIWidgets.ui;
 
 namespace Unity.UIWidgets.gestures {
-    
     public delegate void PointerHoverEventListener(PointerHoverEvent evt);
 
     public delegate void PointerEnterEventListener(PointerEnterEvent evt);
 
     public delegate void PointerExitEventListener(PointerExitEvent evt);
-    
-    
+
+#if UNITY_EDITOR
```

Comment:
you should use #if UNITY_EDITOR/#endif in every necessary places. You may switch the build platform to iOS or Android and then fix the outcoming compile errors to ensure this